### PR TITLE
chore: add release notify action

### PR DESCRIPTION
**Pull request by NiklasMerz**
### Motivation and Context
We want to test a new action that closes issues that are just asking for a new release and reduce noise. Let's start with this plugin.

### Description

This PR adds a new Github action built by me to add a comment and closes the issue if it looks like the issue is just created to ask for a release.

### Checklist

- [ ] I've run the tests to see all new and existing tests pass
- [ ] I added automated test coverage as appropriate for this change
- [x] Commit is prefixed with `(platform)` if this change only applies to one platform (e.g. `(android)`)
- [ ] If this Pull Request resolves an issue, I linked to the issue in the text above (and used the correct [keyword to close issues using keywords](https://help.github.com/articles/closing-issues-using-keywords/))
- [ ] I've updated the documentation if necessary

### CODE REVIEW

**Comment by NiklasMerz** [comment]
Some discussion here: https://github.com/NiklasMerz/release-notify/issues/4

File: .github/workflows/release-notify.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    types: [created]
+    
+
+jobs:
+  action-test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: niklasmerz/release-notify@master
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (timbru31) [inline .github/workflows/release-notify.yml:16]: Won't we need infra to add the secret?

Comment (NiklasMerz) [reply .github/workflows/release-notify.yml:16]: https://docs.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token

I did not find an option to disable it in my orgs. Not sure though.

**Comment by erisu** [comment]
👍

**Pull request merged**